elasticsearch-model/test/integration/mongoid_import_test.rb

```diff
@@ -0,0 +1,155 @@
+require 'test_helper'
+
+begin
+  require 'mongoid'
+  session = Moped::Connection.new("localhost", 27017, 0.5)
+  session.connect
+  ENV["MONGODB_AVAILABLE"] = 'yes'
+rescue LoadError, Moped::Errors::ConnectionFailure => e
+  $stderr.puts "MongoDB not installed or running: #{e}"
+end
+
+if ENV["MONGODB_AVAILABLE"]
+  $stderr.puts "Mongoid #{Mongoid::VERSION}", '-'*80
+
+  logger = ::Logger.new($stderr)
+  logger.formatter = lambda { |s, d, p, m| " #{m.ansi(:faint, :cyan)}\n" }
+  logger.level = ::Logger::DEBUG
+
+  Mongoid.logger = logger unless ENV['QUIET']
+  Moped.logger   = logger unless ENV['QUIET']
+
+  Mongoid.connect_to 'mongoid_articles'
+
+  if defined?(BSON::ObjectId)
+    class BSON::ObjectId
+      def as_json(*args)
+        to_s
+      end
+
+      def to_json(*args)
+        as_json().to_json
+      end
+    end
+  end
+
+  if defined?(Moped::BSON::ObjectId)
+    class Moped::BSON::ObjectId
+      def as_json(*args)
+        to_s
+      end
+
+      def to_json(*args)
+        as_json().to_json
+      end
+    end
+  end
+
+  module Elasticsearch
+    module Model
+      class MongoidImportIntegrationTest < Elasticsearch::Test::IntegrationTestCase
+
+        context "Mongoid importing" do
+          setup do
+            class ::MongoidImportArticle
+              include Mongoid::Document
+              include Elasticsearch::Model
+
+              field :title,      type: String
+              field :views,      type: Integer
+              field :numeric,    type: String  # For the sake of invalid data sent to Elasticsearch
```

Comment:
Why set it up as a `String`? We can always pass a string even with the `field` set? Or will Mongoid cast it to number?